terraspace/app/stacks/artifactory/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
```

Comment:
The name of this stack is a little odd when you come to realize that there is an existing [product](https://jfrog.com/artifactory/) with the same name.

Reply:
Another name suggestion?